docs/faq/modeling.md

```diff
@@ -259,6 +259,26 @@ slots:
 
 See [dynamic enums](https://linkml.io/linkml/schemas/enums.html#dynamic-enums) for more details.
 
+### Are CURIEs used in schema definitions checked for expandability and resolution?  
+
+No, not at this time.  However, linkml_runtime does have methods to help you expand the CURIEs in your data
+using the prefixes in your model (see: linkml_runtime.utils.namespaces.py) into URIs and then checking them is 
+a simple bit of python like [this](https://github.com/biolink/biolink-model/blob/master/scripts/verify_infores.py).
```

Comment:
it's not totally clear how to adapt this - maybe link to curies library?

Reply:
👍 . removed.